typescript/infra/src/deployment/deploy.ts

```diff
@@ -122,15 +123,18 @@ export async function writeAgentConfig(
   environment: DeployEnvironment,
 ) {
   const addresses = getAddresses(environment, Modules.CORE);
-  const addressesForEnv = objFilter(
+  const ethereumAddressesForEnv = objFilter(
```

Comment:
i know it's just renaming something that was there already, but how do we know it's always eth addresses only

afaict we don't actually filter on protocol type in the multiprovider, so if we want only ethereum ones we should filter for `tryGetProtocol` being `ProtocolType.Ethereum`

Reply:
the `isEthereumProtocolChain` import seems unused

Reply:
oops this is a relic of a path I went down that I abandoned, will undo this all